.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: 'Report an unexpected behavior '
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Where are you using Cairo Native**
+In which project are you using Cairo Native
+
+**Version**
+What Cairo Native commit are you using
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+* What project are you running
+* Which Cairo, Sierra, or Starknet transaction are you running
+* What Error are you getting
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. MacOS, Ubuntu 22.04 LTS]
```

Comment:
They should also provide their architecture (x86_64 or aarch64), not only their OS since we have quite a bit of architecture-dependent code.

Reply:
Good!
done